tests/test_relations.py

```diff
@@ -242,6 +255,8 @@ def test_relation_calculation(self):
         self.assertEqual(new_relation.status_code, 201)
         get_relation = self.app.get('/relation/1')
         self.assertEqual(get_relation.status_code, 200)
+        get_specific_relation = self.app.get('/relation/1/6')
+        self.assertEqual(get_specific_relation.status_code, 200)
```

Comment:
Sure, you did positive testing. (returns expected positive response) You also need to do testing for negative responses. (ex. what happens when someone requests for a relation with a relation_id that doesn't exist?)